Resources/public/js/views/ez-dashboardblocksview.js

```diff
@@ -145,7 +145,14 @@ YUI.add('ez-dashboardblocksview', function (Y) {
              * @readOnly
              */
             blocks: {
-                value: [],
+                valueFn: function () {
+                    return [
+                        new Y.eZ.DashboardBlockAllContentView({
+                            priority: 1,
+                            identifier: 'everyones-content'
+                        }).addTarget(this).set('active', this.get('active'))
```

Comment:
you should write:

```js
new Y.eZ.DashboardBlockAllContentView({
    priority: 1,
    bubbleTargets: this,
});
```

the `identifier` is read only (and should be set in the code of `eZ.DashboardBlockAllContentView` itself. It's useless to forward the `active` flag, we know it's not active and there's an event handler that takes care of keeping active up to date.
it's not needed to set the active flag, we know the view is not active yet